### ISSUE TITLE
Packaging for macOS

### ISSUE THREAD

**Issue by mp4096** [mp4096/indentex#32]
The most obvious choice for macOS packaging is Homebrew. We can use [`ripgrep`](https://github.com/BurntSushi/ripgrep/blob/a515c4d601e449dd78441e7f7e14ab0c2299ac02/pkg/brew/ripgrep-bin.rb) as an example.

**Comment by mp4096** [comment]
@syxolk Do you know some Mac users?

@mherb Could we use you as tester when we have the Homebrew package?

**Comment by mherb** [comment]
Sure, I can also help out with Homebrew if you need any help there.

**Comment by mp4096** [comment]
I'll take it.

**Comment by mp4096** [comment]
References:
* https://github.com/Homebrew/brew/blob/master/docs/How-to-Create-and-Maintain-a-Tap.md
* https://github.com/Homebrew/brew/blob/master/docs/Formula-Cookbook.md

**Comment by mp4096** [comment]
Current status: I've made a small draft of a Homebrew tap, now I have to:
- [ ] setup compilation on macOS
- [ ] packaging into a tarball
- [ ] hash digest generation ([`shasum`](https://developer.apple.com/legacy/library/documentation/Darwin/Reference/ManPages/man1/shasum.1.html))
- [ ] configure deployment to GH Releases

**Comment by syxolk** [comment]
Any progress?

**Comment by mp4096** [comment]
Nope, nothing so far. However, come to think about it, we now have a macOS user nearby — say hello to @hannafischer
